{"uid":"51d6c4345e75190e","name":"test_favourites['Сало в соевом соусе вареное']","fullName":"tests.test_favorites_page.TestFavourites#test_favourites","historyId":"8a4687c9d5e006b96dceae3ef31fa528","time":{"start":1689186460356,"stop":1689186460356,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c9eec4a4e3 <unknown>\n#1 0x55c9ee979c76 <unknown>\n#2 0x55c9ee9a2d78 <unknown>\n#3 0x55c9ee99f029 <unknown>\n#4 0x55c9ee9ddccc <unknown>\n#5 0x55c9ee9dd47f <unknown>\n#6 0x55c9ee9d4de3 <unknown>\n#7 0x55c9ee9aa2dd <unknown>\n#8 0x55c9ee9ab34e <unknown>\n#9 0x55c9eec0a3e4 <unknown>\n#10 0x55c9eec0e3d7 <unknown>\n#11 0x55c9eec18b20 <unknown>\n#12 0x55c9eec0f023 <unknown>\n#13 0x55c9eebdd1aa <unknown>\n#14 0x55c9eec336b8 <unknown>\n#15 0x55c9eec33847 <unknown>\n#16 0x55c9eec43243 <unknown>\n#17 0x7fa48c12cb43 <unknown>","statusTrace":"E   selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n      (unknown error: DevToolsActivePort file doesn't exist)\n      (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n    Stacktrace:\n    #0 0x55c9eec4a4e3 <unknown>\n    #1 0x55c9ee979c76 <unknown>\n    #2 0x55c9ee9a2d78 <unknown>\n    #3 0x55c9ee99f029 <unknown>\n    #4 0x55c9ee9ddccc <unknown>\n    #5 0x55c9ee9dd47f <unknown>\n    #6 0x55c9ee9d4de3 <unknown>\n    #7 0x55c9ee9aa2dd <unknown>\n    #8 0x55c9ee9ab34e <unknown>\n    #9 0x55c9eec0a3e4 <unknown>\n    #10 0x55c9eec0e3d7 <unknown>\n    #11 0x55c9eec18b20 <unknown>\n    #12 0x55c9eec0f023 <unknown>\n    #13 0x55c9eebdd1aa <unknown>\n    #14 0x55c9eec336b8 <unknown>\n    #15 0x55c9eec33847 <unknown>\n    #16 0x55c9eec43243 <unknown>\n    #17 0x7fa48c12cb43 <unknown>","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"browser","time":{"start":1689186460357,"stop":1689186461064,"duration":707},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55c9eec4a4e3 <unknown>\n#1 0x55c9ee979c76 <unknown>\n#2 0x55c9ee9a2d78 <unknown>\n#3 0x55c9ee99f029 <unknown>\n#4 0x55c9ee9ddccc <unknown>\n#5 0x55c9ee9dd47f <unknown>\n#6 0x55c9ee9d4de3 <unknown>\n#7 0x55c9ee9aa2dd <unknown>\n#8 0x55c9ee9ab34e <unknown>\n#9 0x55c9eec0a3e4 <unknown>\n#10 0x55c9eec0e3d7 <unknown>\n#11 0x55c9eec18b20 <unknown>\n#12 0x55c9eec0f023 <unknown>\n#13 0x55c9eebdd1aa <unknown>\n#14 0x55c9eec336b8 <unknown>\n#15 0x55c9eec33847 <unknown>\n#16 0x55c9eec43243 <unknown>\n#17 0x7fa48c12cb43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/runner/work/testing_foodgram_project_react/testing_foodgram_project_react/conftest.py\", line 26, in browser\n    browser = webdriver.Chrome(\n              ^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py\", line 84, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py\", line 104, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"feature","value":"Тесты добавления рецептов в избранное."},{"name":"story","value":"Тест добавления рецептов в избранное"},{"name":"tag","value":"favourites_test(scope='class')"},{"name":"tag","value":"run(order=5)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_favorites_page"},{"name":"subSuite","value":"TestFavourites"},{"name":"host","value":"fv-az256-210"},{"name":"thread","value":"2518-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_favorites_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"data","value":"'Сало в соевом соусе вареное'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=5)","favourites_test(scope='class')"]},"source":"51d6c4345e75190e.json","parameterValues":["'Сало в соевом соусе вареное'"]}